packages/fftw.rb

```diff
@@ -1,37 +1,25 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Fftw < Package
-  description 'FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data'
-  homepage 'http://www.fftw.org/'
-  version '3.3.9'
+class Fftw < Autotools
+  description 'FFTW is a C subroutine library for computing the discrete Fourier transform (DFT) in one or more dimensions, of arbitrary input size, and of both real and complex data (as well as of even/odd data, i.e. the discrete cosine/sine transforms or DCT/DST).'
+  homepage 'https://www.fftw.org/'
+  version '3.3.10'
   license 'GPL-2+'
   compatibility 'all'
-  source_url "http://www.fftw.org/fftw-#{version}.tar.gz"
-  source_sha256 'bf2c7ce40b04ae811af714deb512510cc2c17b9ab9d6ddcf49fe4487eea7af3d'
-  binary_compression 'tar.xz'
+  source_url 'https://fftw.org/fftw-3.3.10.tar.gz'
```

Comment:
Keeping `#{version}` makes it a bit easier to do updates, actually.

Reply:
That said, if we include `-py3.12` in the version then we would need to revert back to `@_ver` or use `#{version.split('-').first}`.